src/components/ContentHeader.vue

```diff
@@ -1,72 +1,62 @@
 <template>
   <div>
-    <div class="usa-overlay"></div>
 <header class="usa-header usa-header--basic">
   <div class="usa-nav-container">
-    <div class="usa-navbar">
-      <div class="usa-logo" id="basic-logo">
-        <em class="usa-logo__text">
-          <a href="javascript:void(0)" title="USGS Vizlab">
-            USGS Vizlab
-          </a>
-        </em>
-      </div>
-      <button class="usa-menu-btn">Menu</button>
-    </div>
     <nav aria-label="Primary navigation" class="usa-nav">
-      <button class="usa-nav__close">
-        <img src="/assets/img/usa-icons/close.svg" role="img" alt="Close" />
-      </button>
       <ul class="usa-nav__primary usa-accordion">
+         <li class="usa-nav__primary-item">
+          <button
+            class="usa-accordion__button usa-nav__link"
+            aria-expanded="false"
+            aria-controls="basic-nav-new"
+            onClick="document.getElementById('viz-new').scrollIntoView();"
```

Comment:
I think this needs to be `onClick="document.getElementById('viz-latest').scrollIntoView();"` no element with the id `viz-new` exists

Reply:
Interestingly, once you make that change, and also make that button a `usa-nav__button` instead of a `usa-accordian__button`, the `scrollIntoView()` for this button (What's new) just works, and doesn't offset the margin. Must be something different about this div set up?? not sure, honestly.